x-pack/plugins/transform/public/alerting/transform_health_rule_type/register_transform_health_rule.ts

```diff
@@ -61,24 +61,24 @@ export function getTransformHealthRuleType(): RuleTypeModel<TransformHealthRuleP
     defaultActionMessage: i18n.translate(
       'xpack.transform.alertTypes.transformHealth.defaultActionMessage',
       {
-        defaultMessage: `[\\{\\{rule.name\\}\\}] Transform health check result:
-\\{\\{context.message\\}\\}
-\\{\\{#context.results\\}\\}
-  Transform ID: \\{\\{transform_id\\}\\}
-  \\{\\{#description\\}\\}Transform description: \\{\\{description\\}\\}
-  \\{\\{/description\\}\\}\\{\\{#transform_state\\}\\}Transform state: \\{\\{transform_state\\}\\}
-  \\{\\{/transform_state\\}\\}\\{\\{#health_status\\}\\}Transform health status: \\{\\{health_status\\}\\}
-  \\{\\{/health_status\\}\\}\\{\\{#issues\\}\\}Issue: \\{\\{issue\\}\\}
-  Issue count: \\{\\{count\\}\\}
-  \\{\\{#details\\}\\}Issue details: \\{\\{details\\}\\}
-  \\{\\{/details\\}\\}\\{\\{#first_occurrence\\}\\}First occurrence: \\{\\{first_occurrence\\}\\}
-  \\{\\{/first_occurrence\\}\\}
-  \\{\\{/issues\\}\\}\\{\\{#failure_reason\\}\\}Failure reason: \\{\\{failure_reason\\}\\}
-  \\{\\{/failure_reason\\}\\}\\{\\{#notification_message\\}\\}Notification message: \\{\\{notification_message\\}\\}
-  \\{\\{/notification_message\\}\\}\\{\\{#node_name\\}\\}Node name: \\{\\{node_name\\}\\}
-  \\{\\{/node_name\\}\\}\\{\\{#timestamp\\}\\}Timestamp: \\{\\{timestamp\\}\\}
-  \\{\\{/timestamp\\}\\}
-\\{\\{/context.results\\}\\}
+        defaultMessage: `['{{rule.name}}'] Transform health check result:
+'{{context.message}}'
+'{{#context.results}}'
+  Transform ID: '{{transform_id}}'
+  '{{#description}}'Transform description: '{{description}}'
+  '{{/description}}''{{#transform_state}}'Transform state: '{{transform_state}}'
+  '{{/transform_state}}''{{#health_status}}'Transform health status: '{{health_status}}'
+  '{{/health_status}}''{{#issues}}'Issue: '{{issue}}'
+  Issue count: '{{count}}'
+  '{{#details}}'Issue details: '{{details}}'
+  '{{/details}}''{{#first_occurrence}}'First occurrence: '{{first_occurrence}}'
+  '{{/first_occurrence}}'
+  '{{/issues}}''{{#failure_reason}}'Failure reason: '{{failure_reason}}'
+  '{{/failure_reason}}''{{#notification_message}}'Notification message: '{{notification_message}}'
+  '{{/notification_message}}''{{#node_name}}'Node name: '{{node_name}}'
+  '{{/node_name}}''{{#timestamp}}'Timestamp: '{{timestamp}}'
+  '{{/timestamp}}'
+'{{/context.results}}'
```

Comment:
<img width="1263" alt="image" src="https://github.com/elastic/kibana/assets/5236598/bc70f109-c1b2-4d81-965c-af1b5d3d2dd8">

also with transform rule

Reply:
@darnautov fixed. These will be automatically caught by the `i18n_check` script once the next PR is merged. For now I ran it against the code and fixed these issues. please take another look 🙏🏽